### ISSUE TITLE
Wrong results for translate:false fields in plugins in multi language site

### ISSUE THREAD

**Issue by andreasotto** [getkirby/kirby#2610]
When in multi-language setup and when defined `translate: false` fields in users blueprints, usually the default languages' content is returned as fallback. That is the expected behavior.

That works as long as the `$user` object is not updated. 

To reproduce setup a plugin like this:

```
Kirby::plugin('test/test', [
    'pageMethods' => [
        'myTest' => function () {
             $this->kirby()->user()->update([
                 'testField' => "test"
             ]);
             dump($this->kirby()->user());
        }
    ]
]);
```

When you switch from the default language to one of the secondary languages, the `content` Array from the user object shows empty fields. I would expect the default content from the default language. The fields are defined as `translate: false` in the users blueprint.
If i comment out the update call, the fields are **not** empty and it works as expected.

**Comment by afbora** [comment]
This issue applies to the entire content models not only users. 
Since non-translatable fields are not in the content file, default values correctly fill. 
However, since the non-translatable fields are [set to null](https://github.com/getkirby/kirby/blob/3.4.0/src/Cms/ModelWithContent.php#L564) while saving the content, the `null` data overwrite the default value on [array_merge()](https://github.com/getkirby/kirby/blob/3.4.0/src/Cms/ContentTranslation.php#L103) method.

These are issue tips, but I have no idea about the solution.

Also you can reproduce the with following snippet on secondary language with `text` field `translate: false`

````php
$rand = rand();
dump($page->text());
dump($rand);
$page = $page->update(['text' => $rand]);
dump($page->text());

// old page value: 100
// random: 999
// new page value: null
````

**Comment by lukasbestle** [comment]
I think the issue is that the [`ContentTranslation` object](https://github.com/getkirby/kirby/blob/3.4.0/src/Cms/ModelWithContent.php#L549) does not get cloned. That line should probably be:

```php
$translation = $clone->translation($languageCode)->clone();
```

...otherwise the updates are performed with the existing object, which is still referenced by the old `Page` object (the `clone()` of the `Page` object does *not* also clone the `ContentTranslation` object).

But the question is: Shouldn't the `ContentTranslation` object be immutable like other `Cms` objects? Pinging @bastianallgeier.

**Comment by afbora** [comment]
> $translation = $clone->translation($languageCode)->clone();

Works like a magic 🎉

**Comment by lukasbestle** [comment]
@afbora Awesome, thanks for testing. We should probably wait for Bastian though.

**Comment by bastianallgeier** [comment]
@lukasbestle you are right. The Page::clone() method should actually take care of this. There's a property cache for $translations and that's why the ContentTranslation object is still the old one.

**Comment by lukasbestle** [comment]
Yes, that would be the way to go! Much cleaner than handling it manually afterwards.

As the translations are handled by `ModelWithContent`, the cloning of the `ContentTranslations` objects should happen in a new `ModelWithContent::clone()` method.

**Comment by lukasbestle** [comment]
Seems like we have just forgotten about this, haven't we? We already have an idea for a solution and that solution doesn't look too complex.

**Comment by intellent** [comment]
Hey everybody,

since this issue is on hold for quite a while now, I wonder if there is a workaround. In my case I have a block with a field of type `users` which is set to `translate: false`. In all non-default languages, the block’s custom controller throws an error that the field is `null`.

**Comment by bastianallgeier** [comment]
This issue no longer exists in Kirby 5
